app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -44,13 +44,19 @@ if [ x$ID = xrhel ] || [ x$ID = xcentos ]; then
 
     curl --insecure --output $CONSUMER_RPM <%= subscription_manager_configuration_url %>
     yum localinstall $CONSUMER_RPM -y
-    yum install subscription-manager -y
```

Comment:
This is showing up in multiple PRs now. I think it can be its own PR and easily merged. You can also drop `--insecure` since the URL is HTTP.